wallet-sdk/build.gradle.kts

```diff
@@ -31,13 +32,71 @@ val javadocJar =
     from(dokkaOutputDir)
   }
 
+val sourcesJar by tasks.registering(Jar::class) {
+  archiveClassifier.set("sources")
+  from(kotlin.sourceSets.main.get().kotlin)
+}
+
 publishing {
+  repositories {
+    maven {
+      name = "oss"
+      val releasesRepoUrl = uri("https://oss.sonatype.org/service/local/staging/deploy/maven2/")
+      val snapshotsRepoUrl = uri("https://oss.sonatype.org/content/repositories/snapshots/")
+      url = if (version.toString().endsWith("SNAPSHOT")) snapshotsRepoUrl else releasesRepoUrl
+      credentials {
+        username = System.getenv("OSSRH_USER") ?: return@credentials
+        password = System.getenv("OSSRH_PASSWORD") ?: return@credentials
+      }
+    }
+  }
+
   publications {
     create<MavenPublication>("mavenJava") {
-      groupId = "org.stellar"
+      groupId = "org.stellar.wallet-sdk"
```

Comment:
The `groupId` should be `org.stellar` because the `artifactId` is `wallet-sdk`

Reply:
It's been like this initially, but I thought that if in the future we introduce more packages, it's better if group id doesn't change. I can set it back to org.stellar

Reply:
I used Anchor Platform as an example for `build.gradle` configuration. `group` is set to `org.stellar.anchor-sdk` [here](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/build.gradle.kts#L124). Is that the same one we're using here for `groupId`? I'll let you decide what is the correct value here. :slightly_smiling_face:

Reply:
I will keep it `org.stellar:wallet-sdk:0.1.0` for now
We can later transition to `org.stellar.wallet-sdk:<artifact>:<version>` later